@here/harp-utils/lib/TaskQueue.ts

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Task that can be added to the [[TaskQueue]]
+ */
+export interface Task {
```

Comment:
This interface has a mixture of methods and properties. I think except `execute` all members should be properties

Reply:
They are functions, as they might change until the Task is actually executed, so if i.e. the priority of the Task changes, as i.e. the camera moves and the regarding tile is further away the priority function will be able to generate the updated priority